app/example_app/books.py

```diff
@@ -0,0 +1,112 @@
+from flask import render_template, request, redirect, flash
+from .. import db
+from ..models import  Book, Announcement, Event, Graphic, Borrow
+from . import example_app
+
+@example_app.route("/Employee")
+def employee():
+    return render_template("/books/employee.html")
+
+@example_app.route('/Books', methods=['POST','GET'])
+def books():
+    if request.method == 'POST':
+        nazwa = request.form['nazwa']
```

Comment:
Nie mieszaj języka polskiego i angielskiego.
```suggestion
        name = request.form['name']
```